pyoverkiz/client.py

```diff
@@ -3,6 +3,8 @@
 
```

Comment:
@tetienne lol, see CI. I will make the change to just ignore this, since we won't have time to refactor this huge module near term.

```
  ************* Module pyoverkiz.client
  pyoverkiz/client.py:1:0: C0302: Too many lines in module (1006/1000) (too-many-lines)`
```

Reply:
Well I have my pr where I did the refactor but we didn't test it 😅